src/SDKs/Search/DataPlane/Search.Tests/Tests/ExtensibleEnumTests.cs

```diff
@@ -69,6 +72,23 @@ public void CannotCompareDifferentEnumsForEquality()
             Assert.False(TestEnum.A.Equals((object)TestEnum2.A));
         }
 
+        [Fact]
+        public void CanCreateNonObsoleteTypeFromValue()
+        {
+            TestEnumWithObsolete notObsolete = TestEnumWithObsolete.Create("NotObsolete");
+            Assert.Same(TestEnumWithObsolete.NotObsolete, notObsolete);
+        }
+
+        [Fact]
+        public void CannotCreateObsoleteTypeFromValue()
```

Comment:
The test above was for that... you don’t think that accurately tests what I want?

Reply:
@arv100kri Sorry, I missed that before. It's a bit hard to parse since "NotObsolete" maps to the enum member `Obsolete`. Maybe some renaming would make this more obvious.

Reply:
Naming is hard... I've updated the values and certain property names, let me know what you think?